package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Sometimes you want to reload your app bundle during app runtime. This package will allow you to do it.",
   "main": "lib/commonjs/index.js",
   "module": "lib/module/index.js",
-  "types": "lib/typescript/src/index.d.ts",
+  "types": "lib/typescript/index.d.ts",
```

Comment:
@lachenmayer 

there is not a path for `lib/typescript/index.d.ts` in this repository anymore. are you sure about that?